### ISSUE TITLE
[BUG] Chinese character problem, can't show right

### ISSUE THREAD

**Issue by zhongqingsong** [ReactiveDrop/reactivedrop_translations#222]
Some Chinese characters cannot be displayed properly in the game. So far I've found two characters:
·
×
The first character is used in the character's name. (like  马丁·路德·金  Martin Luther King)
The second character, MM, is often used in games to represent bullets.(7.62×51mm)
Of course, At present, the character display is abnormal, so I used another symbol to temporarily replace them.But I think this expression is not correct, and it affects the professionalism of translation.
And then, I mention it here.
By the way, I test the file with Game at now, this is the test result picture.
1. The actual display effect of the game, there are abnormal display. 
![Snipaste_2022-08-05_23-43-15](https://user-images.githubusercontent.com/35858818/183113998-da9e16a0-fab0-4022-9972-e42a5abdaba6.png)
![Snipaste_2022-08-05_23-43-33](https://user-images.githubusercontent.com/35858818/183114086-9f0d59b1-55f1-41c2-ba8f-4f3cde67eb10.png)

2.Real game translation text, through the normal software can normally display symbols.
![Snipaste_2022-08-05_23-46-06](https://user-images.githubusercontent.com/35858818/183114148-6c5bba50-c88c-41e7-82a1-16c707d5ca47.png)
![Snipaste_2022-08-05_23-45-39](https://user-images.githubusercontent.com/35858818/183114158-d217d8a2-70a0-4047-bcd5-805f6f4a1978.png)

In this case, do I feel that the font file used by the game lacks corresponding character symbols?

**Comment by BenLubar** [comment]
Weird, they do work on my computer:

![image](https://user-images.githubusercontent.com/4257305/183125114-322550bc-939e-4085-8a99-73aea80dac6a.png)

I did also notice that my menus seem to be in two different fonts for different letters so maybe it's loading system fonts?

![image](https://user-images.githubusercontent.com/4257305/183125267-1b3697b9-e00f-4453-8b25-5a29345866a2.png)

**Comment by zhongqingsong** [comment]
> Weird, they do work on my computer:
> 
> ![image](https://user-images.githubusercontent.com/4257305/183125114-322550bc-939e-4085-8a99-73aea80dac6a.png)
> 
> I did also notice that my menus seem to be in two different fonts for different letters so maybe it's loading system fonts?
> 
> ![image](https://user-images.githubusercontent.com/4257305/183125267-1b3697b9-e00f-4453-8b25-5a29345866a2.png)

Yes, Not all text display problems, such as dialog display is normal.
What is found is that the text displayed on the UI interface is not correct. Like the text of my screenshot, it's the description text of the soldier, the weapon. The problem starts a long time ago, 
I wanted to mention it originally, but it was mentioned in the game update a few days ago, which fixed the Chinese symbol problem. So, I re-envoys, found that there is still a display problem.

**Comment by zhongqingsong** [comment]
I looked at the built-in fonts of RD, and the file sizes are relatively small (all hundreds of kilobytes). If it is a standard Chinese font library, the size will generally be a few M. So, I decided that the biggest xarialuni font is the international font library.

So I tried to view the xarialuni.ttf font, and the Windows system reported an error, indicating that the file could not be previewed. It means that this xarialuni font file is actually invalid on Chinese computers? That's why you think "so maybe it's loading system fonts"?
Then, I manually found a font file and replaced the xar file, and there was no change after entering the game. Because I don't know the source code, maybe what I've done is useless. Anyway, I hope it can help a little bit to troubleshoot the problem.
